exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,11 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if (sentence.split("code").length > 1)
```

Comment:
This would work, however it's a little unclear what the intent is. `split` is used to turn a string to an array based on some specifier, what you're looking for is the `.includes` method

Reply:
yes. I know I was dumb in this question :)